data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
```

Comment:
> So what are our exact decommission dates?

Current target is June 18 for final ACH run on Balanced, and June 25 for initial run on Zipmark.

> BTW, is donation link information preserved? How much who wants to donate to who?

Yes, that's all recorded in the Gratipay database, not with our processing partners.

> Do we have backups of this data?

Yes.